bot/exts/help_channels/_cog.py

```diff
@@ -112,11 +115,13 @@ async def claim_channel(self, message: discord.Message) -> None:
 
         self.bot.stats.incr("help.claimed")
 
-        # Must use a timezone-aware datetime to ensure a correct POSIX timestamp.
-        timestamp = datetime.now(timezone.utc).timestamp()
-        await _caches.claim_times.set(message.channel.id, timestamp)
+        # datetime.timestamp() would assume it's local, despite d.py giving a (naïve) UTC time.
```

Comment:
We should probably just use `naive` here :smile:, which is the more usual spelling nowadays.
```suggestion
        # datetime.timestamp() would assume it's local, despite d.py giving a (naive) UTC time.
```

Reply:
Leave my diacritics out of this. Jokes aside, this spelling is much more common than you might think.

Reply:
personally i would prefer the normal i as i have no idea how to properly type the one with 2 dots.